lib/core/src/lib/viewer/components/viewer.component.spec.ts

```diff
@@ -525,6 +525,34 @@ describe('ViewerComponent', () => {
             });
         });
 
+        describe('Close Button', () => {
+
+            it('should show close button on left side when showCloseButton is true and allowGoBack is true', () => {
+                component.showCloseButton = true;
+                component.allowGoBack = true;
+                fixture.detectChanges();
+
+                expect(element.querySelector('.adf-viewer-close-button')).not.toBeNull();
+
+            });
+
+            it('should show close button on right side when showCloseButton is true and allowGoBack is true', () => {
```

Comment:
and allowGoBack is false in the title.
as you assigning the value component.allowGoBack = false;